torch/onnx/symbolic_helper.py

```diff
@@ -545,7 +545,18 @@ def _is_fp(value):
                 torch.bfloat16,
             )
         else:
-            type = value.type().scalarType()
+            try:
+                type = value.type().scalarType()
```

Comment:
Have you considered using `if ... elif ... ...` to check for `torch.TensorType` and `torch.ListType`, as compared to try-catch?

Reply:
I did, I was unsure whether *all* `ListType` will not return `scalarType` or if there can be `torch.ListType` with assigned `scalarType`. Do you know about that?

Reply:
I will try the `if-else` approach and assume any `_C.ListType` would fail the same